### PR TITLE
add code for pop the item from the stack

### PR DESCRIPTION
**Pull request by YuweAEC**
image

push pop and veiw

### CODE REVIEW

File: daily-code/stack-operation.c [hunk 1]
```diff
@@ -1,72 +1,71 @@
...
-{
-    printf("\n Stack Contains \n");
-    for(int i=t; i>=0; i--)
-    printf("%d \n", s[i]);
-}
 
 void main()
 {
     int op, it, v;
     while (100) //Use an infinite loop to continuously read user input
```
Comment (gauravsinghaec) [inline daily-code/stack-operation.c]: you can write while(true)

File: daily-code/stack-operation.c [hunk 2]
```diff
@@ -1,72 +1,71 @@
...
+            printf("\nEnter the item to push: ");
+            scanf("%d", &it);
             push(it);   /* invoke or calling the push function */
             break;
         case 2:
             v=pop();   /* invoke or calling the pop function */
             if (v>0)
             {
-                printf(" %d is pop from the stack", v);
+                printf("\n%d is pop from the stack", v);
```
Comment (gauravsinghaec) [inline daily-code/stack-operation.c]: `is popped`

**Review by gauravsinghaec — APPROVED** [review]

**Comment by gauravsinghaec** [comment]
Check my comment before merge

File: daily-code/stack-operation.c (hunk 1 above)
Comment (YuweAEC) [reply daily-code/stack-operation.c]: Accha ok

**Comment by YuweAEC** [comment]
> Check my comment before merge 

Yes checked

**Pull request closed without merging**